RestSharp/RestClient.Sync.cs

```diff
@@ -40,6 +40,7 @@ public virtual IRestResponse Execute(IRestRequest request, Method httpMethod)
         {
             if (request == null)
                 throw new ArgumentNullException(nameof(request));
+            request.Method = httpMethod;
```

Comment:
What's the point of this change?

Reply:
This is actually something that I need to fix, before moving forward. When I added that line there was not a line to override the HTTP Method on the request as this method is documented to do on [Line 37](https://github.com/restsharp/RestSharp/blob/29b7a38a3ee2478311f2a7d7c954d56d4a37f914/RestSharp/RestClient.Sync.cs#L37). I was reading through the `RestClient` and studying its structure when I happened to notice that the Method was not being overridden, the `httpMethod` parameter was being passed in but never used.

It appears that there was a merge problem between my `master` and `develop` branches, making it look like there are two lines, so that's what I'm going to fix.

Reply:
Yeah it actually looks like the commit right before this line that I added had done the same thing and I just missed it. So I'll revert that change and push back up.